### PR TITLE
#155738437 Setup Repo

### PR DESCRIPTION
**Pull request by NabunyaLilian**
**What does this PR do**
- Sets up the project repository

**Tasks to be completed**
- Set up IDE (Android Studio)
- Initialise android app folder with git
- Customise color palette

**Relevant PT stories**
[#155738437](https://www.pivotaltracker.com/story/show/155738437)
[#155738456](https://www.pivotaltracker.com/story/show/155738456)

**ScreenShot**
Screen Shot 2019-03-25 at 9 23 32 AM

### CODE REVIEW

File: tools/script-findbugs.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apply plugin: 'findbugs'
```
Comment (johngorithm) [inline tools/script-findbugs.gradle]: This file is not expected in this PR. Kindly remove it

File: tools/rules-pmd.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
```
Comment (johngorithm) [inline tools/rules-pmd.xml]: This file is not expected in this PR. Kindly remove it

File: tools/rules-findbugs.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<FindBugsFilter>
```
Comment (johngorithm) [inline tools/rules-findbugs.xml]: This file is not expected in this PR. Kindly remove it

File: tools/rules-checkstyle.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
```
Comment (johngorithm) [inline tools/rules-checkstyle.xml]: This file is not expected in this PR. Kindly remove it

File: tools/pmd.gradle [hunk 5]
```diff
@@ -0,0 +1,16 @@
+apply plugin: 'pmd'
```
Comment (johngorithm) [inline tools/pmd.gradle]: This file is not expected in this PR. Kindly remove it

File: tools/checkstyle.gradle [hunk 6]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (johngorithm) [inline tools/checkstyle.gradle]: This file is not expected in this PR. Kindly remove it

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">My Application</string>
```
Comment (johngorithm) [inline app/src/main/res/values/strings.xml]: Change this to reflect your application's name. Be creative with that 👍

File: app/src/main/java/com/example/thecodelab/MainActivity.java [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+
+    private void someMethod(int variable) {
```
Comment (johngorithm) [inline app/src/main/java/com/example/thecodelab/MainActivity.java]: Remove this redundant method

File: .circleci/config.yml [hunk 9]
```diff
@@ -0,0 +1,30 @@
+version: 2
```
Comment (johngorithm) [inline .circleci/config.yml]: This file is not expected in this PR. Kindly remove it

**Review by johngorithm — CHANGES_REQUESTED** [review]
Good Job @NabunyaLilian 
Kindly attend the reviews below and equally ensure to `gitignore` the contents of `.idea/` directory.

Learn about what to ignore in an Android project.
[Useful Link](https://proandroiddev.com/deep-dive-into-idea-folder-in-android-studio-53f867cf7b70)

**Comment by NabunyaLilian** [comment]
thanks for the feedback @johngorithm, I have worked on it.

File: .gitignore [hunk 10]
```diff
@@ -63,3 +63,8 @@ fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
 fastlane/readme.md
+
+.DS_Store
+
+tools/
```
Comment (johngorithm) [inline .gitignore]: 👁 👁

**Review by johngorithm — APPROVED** [review]
LGTM

**Review by joshNic — APPROVED** [review]
LGTM am merging this

**Pull request merged**